server/dbutil/dbutil.go

```diff
@@ -26,13 +26,13 @@ const (
 
 // DB groups a sql.DB and the driver used to initialize it
 type DB struct {
-	sqlDB  *sql.DB
+	SQLDB  *sql.DB
```

Comment:
what do you think about composition here?
It would allow not write `db.SQLDB.method` everywhere, just `db.method`

Reply:
Done in 77f23a1